test/JDBC/expected/BABEL_4367.out

```diff
@@ -246,10 +246,99 @@ varchar
 ~~END~~
 
 
+-- #### TYPE  #####
+CREATE TYPE BABEL_4367_TYPE_1 from datetime
+GO
+
+DROP TYPE BABEL_4367_TYPE_1, BABEL_4367_TYPE_2
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: syntax error near ',' at line 1 and character position 27)~~
+
+DROP TYPE IF EXISTS BABEL_4367_TYPE_1, BABEL_4367_TYPE_2
```

Comment:
why is there syntax error, atleast PG allows it

Reply:
SQL server does not allow multiple TYPE drop